### PR TITLE
Dev #19685: PHP 8.3 compatibility - LS Core 

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/core/plugins/TwoFactorAdminLogin/helper/phpqrcode.php [hunk 1]
```diff
@@ -157,7 +157,9 @@ public static function set(&$srctab, $x, $y, $repl, $replLen = false) {
  */
 
     class QRtools {
-    
+
+        public $frames;
```
Comment (Shnoulle) [inline application/core/plugins/TwoFactorAdminLogin/helper/phpqrcode.php:161]: Need to separate this one ?

See https://bugs.limesurvey.org/view.php?id=19686

**Review by Shnoulle — APPROVED** [review]
The code on core seems OK.
More mantis issue updated than just core issue

File: application/core/plugins/TwoFactorAdminLogin/helper/phpqrcode.php (hunk 1 above)
Comment (gabrieljenik) [reply application/core/plugins/TwoFactorAdminLogin/helper/phpqrcode.php:161]: Yep, should have been on that ticket. But let's leave it as it is

**Comment by gabrieljenik** [comment]
It was decided that this needs to be merged into master not develop.
So closing the PR in favor of https://github.com/LimeSurvey/LimeSurvey/pull/3962

**Pull request closed without merging**